packages/patternfly-4/react-charts/src/components/ChartDonutUtilization/ChartDonutUtilization.tsx

```diff
@@ -496,11 +509,6 @@ export const ChartDonutUtilization: React.FunctionComponent<ChartDonutUtilizatio
     return computedData;
   };
 
-  const getData = () => {
-    const datum = [{ ...data }];
-    return Data.formatData(datum, { x, y, ...rest }, ['x', 'y']).sort((a: any, b: any) => a._y - b._y);
-  };
-
```

Comment:
Why are functions being moved? They were in alphabetical order

Reply:
Reverted to original order